### PR TITLE
[v2.0.3] Issue 3115 fixes

### PR DESCRIPTION
**Pull request by chiranjibi009**
This PR contains the issue fixes for issue #3115 

For this issue fixes 

- A new column(`contact_email`) is introduced to capture contact email address once a new study is published from Study Builder, so that while inviting participant `contact_email` column can be used as a contact email address for that particular study.

### CODE REVIEW

File: participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/dao/StudiesDaoImpl.java [hunk 1]
```diff
@@ -93,6 +93,7 @@ public ErrorBean saveStudyMetadata(StudyMetadataBean studyMetadataBean) {
       studyInfo.setModifiedBy(String.valueOf(0));
       studyInfo.setModified(Timestamp.from(Instant.now()));
       studyInfo.setLogoImageUrl(studyMetadataBean.getLogoImageUrl());
+      studyInfo.setContactEmail(studyMetadataBean.getContactEmail());
```
Comment (moschetti) [inline participant-datastore/user-mgmt-module/user-mgmt/src/main/java/com/google/cloud/healthcare/fdamystudies/dao/StudiesDaoImpl.java:96]: Please update file copyright to `2020-2021`

File: participant-datastore/user-mgmt-module/user-mgmt/src/test/java/com/google/cloud/healthcare/fdamystudies/controller/StudiesControllerTest.java [hunk 2]
```diff
@@ -96,7 +96,8 @@ public StudyMetadataBean createStudyMetadataBean() {
         Constants.APP_ID_VALUE,
         Constants.APP_NAME,
         Constants.APP_DESCRIPTION,
-        Constants.LOGO_IMAGE_URL);
+        Constants.LOGO_IMAGE_URL,
```
Comment (moschetti) [inline participant-datastore/user-mgmt-module/user-mgmt/src/test/java/com/google/cloud/healthcare/fdamystudies/controller/StudiesControllerTest.java:99]: Please update file copyright to `2020-2021`

File: participant-datastore/user-mgmt-module/user-mgmt/src/test/java/com/google/cloud/healthcare/fdamystudies/testutils/Constants.java [hunk 3]
```diff
@@ -70,4 +70,5 @@ private Constants() {}
   public static final String STUDY_INFO_ID = "2";
   public static final String STUDY_ID_1 = "study-1";
   public static final String USER_EMAIL = "cdash93@gmail.com";
+  public static final String CONTACT_EMAIL_ID = "contactemail@gmail.com";
```
Comment (moschetti) [inline participant-datastore/user-mgmt-module/user-mgmt/src/test/java/com/google/cloud/healthcare/fdamystudies/testutils/Constants.java:73]: Please update file copyright to `2020-2021`

File: response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/bean/StudyMetadataBean.java [hunk 4]
```diff
@@ -36,4 +36,5 @@
   private String appName;
   private String appDescription;
   private String logoImageUrl;
+  private String contactEmail;
```
Comment (moschetti) [inline response-datastore/response-server-service/src/main/java/com/google/cloud/healthcare/fdamystudies/bean/StudyMetadataBean.java:39]: Please update file copyright to `2020-2021`

**Comment by chiranjibi009** [comment]
@moschetti Copy right issue has been fixed. Please review the PR.

**Review by moschetti — APPROVED** [review]

**Pull request merged**